pyomo/core/expr/visitor.py

```diff
@@ -1448,15 +1484,15 @@ def visiting_potential_leaf(self, node):
 
 
 def _expression_is_fixed(node):
-    """
-    Return the polynomial degree of the expression.
+    """Return bool indicating if this expression is fixed (non-variable)
 
     Args:
         node: The root node of an expression tree.
 
     Returns:
         A non-negative integer that is the polynomial
         degree if the expression is polynomial, or :const:`None` otherwise.
```

Comment:
This doesn't match line 1487 above. Looks like this was a copy-paste error from long ago?

Reply:
fixed.